website/docs/docs/get-started/dbt-cloud-features.md

```diff
@@ -17,9 +17,9 @@ With the Cloud IDE, you can:
 - Generate and view documentation of your dbt project
 - Leverage [git and version-control your code](/docs/collaborate/git/version-control-basics) from your browser with a couple of clicks
 - Create and test Python models:
-    * You can `compile` Python models to see the full function that gets executed in your data platform
-    * You can see Python models in DAG in dbt version 1.3 and higher
-    * You can't preview python models, currently
+    * Compile Python models to see the full function that gets executed in your data platform
+    * See Python models in DAG in dbt version 1.3 and higher
+    * Currently you can't preview python models
```

Comment:
```suggestion
    * Currently, you can't preview python models
```

Reply:
omg i couldve sworn i already fixed this with a comma 🤦‍♀️ i must've not saved. thank you @matthewshaver !